src/irmin-pack/unix/import.ml

```diff
@@ -55,3 +55,11 @@ module Result_syntax = struct
   let ( let+ ) res f = Result.map f res
   let ( let* ) res f = Result.bind res f
 end
+
+type int_bigarray = (int, Bigarray.int_elt, Bigarray.c_layout) Bigarray.Array1.t
+(** [int_bigarray] is the raw type for the mapping file data, exposed via mmap;
+    this type abbreviation is used in several places throughout irmin-pack.unix,
+    so included here to avoid repetition *)
+
```

Comment:
```suggestion
*)
```
since this is the case for almost everything in import, there is no point in saying it here too?

Reply:
Agreed. Will push shortly.